docs/howto/update_major_package_spec.md

```diff
@@ -0,0 +1,61 @@
+# Update between Package Spec major versions
+
+Each Package Spec major version may bring breaking changes or additional
+validations that may make tests to fail for some packages. Find here guidelines
+about how to fix these issues.
+
+Version of the Package Spec used by a package is defined by the `format_version`
+setting in the `manifest.yml` file.
+
+## Upgrading from Package Spec v1 to v2
```

Comment:
nit: I would add a prefix "Troubleshooting" as this is rather information about how to fix an issue.